src/execution.rs

```diff
@@ -458,6 +479,37 @@ impl Print {
     }
 }
 
+impl Conditional {
+    fn execute(&self, exec: &mut ExecutionContext) -> Result<(), ExecutionError> {
+        for arm in &self.arms {
+            if arm.condition.evaluate(exec)?.into_bool(exec.graph)? {
+                for stmt in &arm.statements {
+                    stmt.execute(exec)?;
+                }
+                return Ok(());
```

Comment:
Unless I'm mistaken, this could also just be `break` (which might make it more clear that only one arm is ever executed).